settings.gradle

```diff
@@ -30,3 +30,4 @@ include 'patchwork-networking'
 include 'patchwork-networking-messages'
 include 'patchwork-vanilla-patches'
 include 'patchwork-enum-hacks'
+include 'patchwork-extensions-block'
```

Comment:
This is supposed to be alphabetical. Can you fix enum-hacks while you're at it?

Reply:
none of this seems particularly alphabetical at all, considering "gui" and "networking" come after "registries"

Reply:
well it *should* be alphabetical, but yeah that should probably be fixed after the PR is merged